asm/istio/istio-operator.yaml

```diff
@@ -26,6 +26,8 @@ spec:
           value: "PROJECT_ID.svc.id.goog|https://storage.googleapis.com/mesh-ca-resources/spiffe_bundle.json" # {"$ref":"#/definitions/io.k8s.cli.substitutions.spiffe-bundle-endpoints"}
         - name: ENABLE_STACKDRIVER_MONITORING
           value: "true" # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.controlplane.monitoring.enabled"}
+        - name: TOKEN_AUDIENCES
+          value: "PROJECT_ID.svc.id.goog" # {"$ref":"#/definitions/io.k8s.cli.substitutions.token-audiences"}
   meshConfig:
```

Comment:
which issue did we fix with TOKEN_AUDIENCES? This was not required for 1.7.

Reply:
it may break multi-project service mesh installation.

Reply:
the security tests failed without the TOKEN_AUDIENCE for asm1.8